e2e/api/steps.js

```diff
@@ -289,7 +303,89 @@ module.exports = {
     await assertCorrectEventsAreAvailableToUser(config.defendantSolicitorUser, 'AWAITING_RESPONDENT_ACKNOWLEDGEMENT');
     await assertCorrectEventsAreAvailableToUser(config.adminUser, 'AWAITING_RESPONDENT_ACKNOWLEDGEMENT');
 
+    //removed because it's not needed for the further tests
+    deleteCaseFields('respondent1Copy');
+    deleteCaseFields('respondent2Copy');
+    deleteCaseFields('solicitorReferencesCopy');
+  },
+
+  acknowledgeClaimSolicitorOne: async (user) => {
+    await apiRequest.setupTokens(user);
+
+    eventName = 'ACKNOWLEDGE_CLAIM';
+    let returnedCaseData = await apiRequest.startEvent(eventName, caseId);
+
+    //because sol one cannot see respondent 2 details
+    deleteCaseFields('respondent2');
+
+    assertContainsPopulatedFields(returnedCaseData);
+    caseData = returnedCaseData;
+
+    deleteCaseFields('systemGeneratedCaseDocuments');
+    deleteCaseFields('solicitorReferences');
+    deleteCaseFields('solicitorReferencesCopy');
+    deleteCaseFields('respondentSolicitor2Reference');
+
+    await validateEventPages(data.ACKNOWLEDGE_CLAIM_SOLICITOR_ONE);
+
+    await assertError('ConfirmNameAddress', data[eventName].invalid.ConfirmDetails.futureDateOfBirth,
+      'The date entered cannot be in the future');
+
+    await assertSubmittedEvent('AWAITING_RESPONDENT_ACKNOWLEDGEMENT', {
+      header: '',
+      body: ''
+    });
+
+    await waitForFinishedBusinessProcess(caseId);
+    await assertCorrectEventsAreAvailableToUser(config.applicantSolicitorUser, 'AWAITING_RESPONDENT_ACKNOWLEDGEMENT');
+    await assertCorrectEventsAreAvailableToUser(config.defendantSolicitorUser, 'AWAITING_RESPONDENT_ACKNOWLEDGEMENT');
+    await assertCorrectEventsAreAvailableToUser(config.adminUser, 'AWAITING_RESPONDENT_ACKNOWLEDGEMENT');
+
+    //removed because it's not needed for the further tests
+    //might fail if 1v1, need to check
```

Comment:
this doesn't fail, i've double checked. i forgot to remove this comment, I'll remove it in my next commit.